<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X31JzForm</class>
 <widget class="QWidget" name="X31JzForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lbStart">
          <property name="styleSheet">
           <string notr="true">border-image:none;</string>
          </property>
          <property name="text">
           <string>S:18：01：13</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbEnd">
          <property name="styleSheet">
           <string notr="true">border-image:none;</string>
          </property>
          <property name="text">
           <string>E:18：01：13</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="styleSheet">
         <string notr="true">border-image:none;</string>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbRunningTime">
        <property name="styleSheet">
         <string notr="true">border-image:none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/destop/start.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/destop/stop.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>122</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rbActive">
      <property name="styleSheet">
       <string notr="true">border-image:none;</string>
      </property>
      <property name="text">
       <string>主动</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbPassive">
      <property name="styleSheet">
       <string notr="true">border-image:none;</string>
      </property>
      <property name="text">
       <string>被动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="hlvContainer">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">QTabBar::tab {
    border-image:url(:/destop/background.png);
    color:#EBECFB;
    padding-right:10px;
    padding-top:5px;
    padding-bottom:5px;
    padding-left:5px;
    border:1px solid #F2F2F2;
}
QTabBar::tab:hover {
    color:#4169E1;
}
QTabBar::tab:selected {
    color:#398AF1;
}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>被动窄带多波束LOFAR</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MStreampColorPlot" name="mbLofarPlot" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>被动宽带警戒</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MStreampColorPlot" name="pwdPlot" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="MStreampColorPlot" name="pwdPlot_2" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>331</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MStreampColorPlot</class>
   <extends>QWidget</extends>
   <header location="global">mstreampcolorplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnStart</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>slider</tabstop>
  <tabstop>rbActive</tabstop>
  <tabstop>rbPassive</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
